doc/algorithms.rst

```diff
@@ -705,7 +708,7 @@ Examples:
 Min-Max
 ^^^^^^^^
 Detailed description of parameters and semantics are described in
-`Intel DAAL Min-Max <https://software.intel.com/en-us/daal-programming-guide-min-max>`__
+`Intel DAAL Min-Max <https://github.com/IntelPython/daal4py/blob/master/examples/normalization_minmax_batch.py>`__
```

Comment:
It must be a link to description but provided the link to example, is not it?

Reply:
It must have been a misclick - I meant to change the broken example link.  give me one moment, I needed to add something else anyways.